src/Tools/dotnet-monitor/Egress/Extension/ExtensionEgressPayload.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Egress
+{
+    public class ExtensionEgressPayload
+    {
+        public EgressArtifactSettings Settings { get; set; }
+        public IDictionary<string, string> Properties { get; set; }
+        public IDictionary<string, string> Configuration { get; set; }
+        public string ProfileName { get; set; }
```

Comment:
`ProviderName` to keep things consistent. In future changes, I would imagine that we'll refer to the fact that egress providers have a name and the word "profile" is not going to appear in any documentation or implementation other than here.